tests/functional/force_auth.js

```diff
@@ -151,13 +151,17 @@ define([
         .then(testAccountNoLongerExistsErrorShown);
     },
 
-    'forgot password flow via force-auth goes directly to confirm email screen': function () {
+    'forgot password flow via force-auth': function () {
       return this.remote
         .then(createUser(email, PASSWORD, { preVerified: true }))
         .then(openForceAuth({ query: { email: email }}))
         .then(click('.reset-password'))
 
-        .then(testElementExists('#fxa-confirm-reset-password-header'))
+        .then(testElementExists('#fxa-reset-password-header'))
+        .then(testElementValueEquals('input[type=email]', email))
+        .then(testElementDisabled('input[type=email]'))
+        .then(testElementTextInclude('.prefillEmail', email))
+
         // user remembers her password, clicks the "sign in" link. They
```

Comment:
Oooh.. Add a second test here to ensure that a user who resets their password and is at the confirm screen, then clicks "sign in" still goes back to force auth.